Library/Homebrew/cask/spec/cask/artifact/uninstall_spec.rb

```diff
@@ -7,12 +7,24 @@
     Hbc::Artifact::Uninstall.new(cask, command: Hbc::FakeSystemCommand)
   }
 
+  let(:test_home_path) { TEST_FIXTURE_DIR }
+  let(:original_home_path) { ENV['HOME'] }
+  let(:absolute_path) { TEST_FIXTURE_DIR/'cask'/'absolute_path' }
+  let(:path_with_tilde) { TEST_FIXTURE_DIR/'cask'/'path_with_tilde' }
+  let(:glob_path1) { TEST_FIXTURE_DIR/'cask'/'glob_path1' }
+  let(:glob_path2) { TEST_FIXTURE_DIR/'cask'/'glob_path2' }
+
   before(:each) do
+    ENV['HOME'] = test_home_path
```

Comment:
To avoid overriding `HOME` you can leave the `absolute_path` and `path_with_tilde` as they were, and only use `TEST_TMPDIR` for the globbed paths.

You still need to `FileUtils.touch` the globbed paths, though.